WebDriverAgentLib/Categories/NSExpression+FBFormat.h

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/*! Notification used to notify about unknown key in predicate expression */
+static NSString *const FBUnknownPredicateKeyException;
```

Comment:
If you make it `static` it will be unusable outside of this file, which is not what you want.